### PR TITLE
Remove VAOS breadcrumb from print view

### PR DESCRIPTION
**Pull request by jenniemc**
## Description
When veteran prints their appointment list or detailed appointment page, the printed page should not include the breadcrumb

## Original issue(s)
department-of-veterans-affairs/va.gov-team#45288

## Testing done
local test

## Screenshots
Screen Shot 2022-08-05 at 3 10 11 PM

## Acceptance criteria
- [ ] print view does not have breadcrumb in the appointment details page or the appointment list queue

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/VideoVisitInstructions.jsx [hunk 1]
```diff
@@ -54,50 +23,58 @@ export function VideoVisitInstructions({ instructionsType }) {
       </div>
     );
   }
+  if (instructionsType === 'Video Visit Preparation') {
+    return (
```
Comment (jenniemc) [inline src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/VideoVisitInstructions.jsx:27]: This is to cleanup bloat from the component file

File: src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/VideoVisitInstructions.jsx [hunk 2]
```diff
@@ -1,41 +1,10 @@
...
-    return videoVisitInstructionsText;
-  }
-
-  return null;
-}
+import NewTabAnchor from '../../../components/NewTabAnchor';
 
 export function VideoVisitInstructions({ instructionsType }) {
-  if (instructionsType === MEDICATION_REVIEW) {
+  if (instructionsType === 'Medication Review') {
```
Comment (vbahinwillit) [inline src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/VideoVisitInstructions.jsx:7]: Why the change? Better to use the defined constant.

File: src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/VideoVisitInstructions.jsx [hunk 3]
```diff
@@ -54,50 +23,58 @@ export function VideoVisitInstructions({ instructionsType }) {
       </div>
     );
   }
+  if (instructionsType === 'Video Visit Preparation') {
```
Comment (vbahinwillit) [inline src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/VideoVisitInstructions.jsx:26]: Same here. Could use VIDEO_VISIT_PREPARATION

File: src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/VideoVisitInstructions.jsx (hunk 2 above)
Comment (jenniemc) [reply src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/VideoVisitInstructions.jsx]: done

File: src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/VideoVisitInstructions.jsx (hunk 3 above)
Comment (jenniemc) [reply src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/VideoVisitInstructions.jsx]: done

**Review by vbahinwillit — APPROVED** [review]
LGTM
